bench/workgen/workgen.cxx

```diff
@@ -1360,30 +1397,38 @@ Thread::describe(std::ostream &os) const
 
 Operation::Operation()
     : _optype(OP_NONE), _internal(nullptr), _table(), _key(), _value(), _config(),
-      transaction(nullptr), _group(nullptr), _repeatgroup(0), _timed(0.0)
+      transaction(nullptr), _group(nullptr), _repeatgroup(0), _timed(0.0), _random_table(false)
```

Comment:
It is not clear to me if a none operation will have a table associated with it and if that would mean random_table false or true.